permits/filters.py

```diff
@@ -61,7 +61,11 @@ class DepartmentPermitRequestFilterSet(BasePermitRequestFilterSet):
         lookup_expr="icontains", label=_("Auteur de la demande"),
     )
     works_object_types__works_object = django_filters.filters.ModelChoiceFilter(
-        queryset=models.WorksObject.objects.order_by("name"),
+        queryset=models.WorksObject.objects.filter(
+            works_object_types__permit_requests__isnull=False
```

Comment:
Is it possible to filter `WorksObjet` visible for the current user (= displayed in the table) instead of all `WorksObject` with data application wide?

Reply:
I will take a look at the logic for retrieving the visible objects for the current user and fix the QuerySet if it is simple enough 👍